Dockerfile

```diff
@@ -18,27 +18,15 @@ RUN apk add --no-cache ca-certificates su-exec
 
 COPY --from=builder /go/src/github.com/syncthing/syncthing/syncthing /bin/syncthing
 
-ENV STNOUPGRADE=1 PUSR=syncthing PUID=1000 PGRP=syncthing PGID=1000
+ENV PUID=1000 PGID=1000
 
 HEALTHCHECK --interval=1m --timeout=10s \
   CMD nc -z localhost 8384 || exit 1
 
-ENTRYPOINT true \
- && ( getent group "${PGRP}" >/dev/null \
-      || addgroup \
-          -g "${PGID}" \
-          "${PGRP}" \
-    ) \
- && ( getent passwd "${PUSR}" >/dev/null \
-      || adduser \
-          -h /var/syncthing \
-          -G "${PGRP}" \
-          -u "${PUID}" \
-          "${PUSR}" \
-    ) \
- && chown "${PUSR}:${PGRP}" /var/syncthing \
- && su-exec "${PUSR}:${PGRP}" \
+ENTRYPOINT \
+  chown "${PUID}:${PGID}" /var/syncthing \
+  && su-exec "${PUID}:${PGID}" \
```

Comment:
Who creates them if they don't exist?
The user can choose to specify whatever he wants.

Reply:
No-one creates them. They don't need to be created, that's the point of the change.

Reply:
I didn't read the text, but I didn't think this was possible.